### PR TITLE
Add more detail on geometry set-up to where's my reader

### LINKED ISSUE #130: Not clear enough that distances need to be set on Y axis in 03_where_is_my_reader
The CIL convention is that distances are set along the Y axis.

In this notebook, for exercise C I tried:

```python
geometry = AcquisitionGeometry.create_Cone2D(source_position=[0,0], detector_position=[source_to_detector_distance, 0], rotation_axis_position=[source_to_object_distance, 0])
```

This gives an error:

```ZeroDivisionError: float division by zero```

I don't think this is at all clear to the user what they have done wrong.

The solution is:
```python

geometry = AcquisitionGeometry.create_Cone2D(source_position=[0,0],
                                             detector_position=[0,source_to_detector_distance],
                                             rotation_axis_position=[0,source_to_object_distance])
```

### PR DESCRIPTION
**Pull request by lauramurgatroyd**
Closes #130

### CODE REVIEW

File: demos/1_Introduction/exercises/03_where_is_my_reader.ipynb [hunk 1]
```diff
@@ -285,7 +285,10 @@
     "\n",
     "You will have to set the position of the source, rotation axis (object) and detector. Look at the [documentation of `create_Cone2D`](https://tomographicimaging.github.io/CIL/nightly/framework/#cone2d-geometry-fanbeam) as well as the notebook [00_CIL_geometry](../00_CIL_geometry.ipynb) to help you.\n",
     "\n",
-    "1. Create a fan-beam geometry using the `AcquisitionGeometry.create_Cone2D`\n",
+    "1. Create a fan-beam geometry using the `AcquisitionGeometry.create_Cone2D`. Note that in CIL, we define the **source to detector distance along the y axis** - be mindful of this when setting up your geometry. E.g. see this illustration of a 3D cone beam geometry, with the axes shown:\n",
```
Comment (hrobarts) [inline demos/1_Introduction/exercises/03_where_is_my_reader.ipynb]: I would maybe say ` we define the **source to object and source to detector distance along the y axis**`

**Review by hrobarts — COMMENTED** [review]
Hi Laura, this looks like a very useful addition to me. Maybe you could also add a comment in the correct code snippet to highlight that the `source_to_detector_distance` and `source_to_object_distance` are in the y axis position?

**Review by gfardell — CHANGES_REQUESTED** [review]
```
1. Create a fan-beam geometry using the `AcquisitionGeometry.create_Cone2D`. Note that in CIL, we define the **source to detector distance along the y axis** - be mindful of this when setting up your geometry. E.g. see this illustration of a 3D cone beam geometry, with the axes shown:
```

I would say our default, i.e. the default system has the rotation axis aligned with the Z axis, the detector rows (Dx) are parallel to the X-axis, and the detector columns (Dy) are parallel to the Z axis. But all this can be set manually. 

For fan-beam they can define the source and the detector where they like, but they'll need to update the detector_direction_x too. We recommend using our defaults as much as possible.

**Comment by lauramurgatroyd** [comment]
Thanks. When I have more time I need to try setting up this example with the distances along a different axis. Last time I tried, no matter what I did it didn't work, but not sure I have the code any more. So yes can update to say they can set it all manually but strongly recommend the default

**Comment by lauramurgatroyd** [comment]
Updated after discussion with @gfardell

File: demos/1_Introduction/exercises/03_where_is_my_reader.ipynb (hunk 1 above)
Comment (lauramurgatroyd) [reply demos/1_Introduction/exercises/03_where_is_my_reader.ipynb]: has been updated now :)

**Review by gfardell — APPROVED** [review]

**Pull request merged**